test/snapshot/helper/conversions.ts

```diff
@@ -89,7 +91,8 @@ export const compareTwoXml = (file1: string, file2: string): Chai.Assertion => {
     ignoreDeclaration: true,
     numberParseOptions: { leadingZeros: false, hex: false },
   });
-  return expect(parser.parse(file1)).to.deep.equal(parser.parse(file2));
+
+  return expect(parser.parse(file1)).to.deep.equalInAnyOrder(parser.parse(file2));
```

Comment:
when comparing xml files, allow for them to "match" if they have array with the same elements out of order.

This means you can receive wrong-order-xml from retrieve or existing projects.  The decompose-recompose trip will "fix" your order (and that's tested with the snapshot test), but the resulting file should be semantically the same if not literally.